src/JAVA_PROGRAMS/Duplicate_Chracter.java

```diff
@@ -0,0 +1,34 @@
+package JAVA_PROGRAMS;
```

Comment:
Correct the program spelling

Regards ,
Lead

Reply:
kk sure

Reply:
go ahead n m merge